docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -0,0 +1,245 @@
+    "                                  trans_primitives=[\"negate\"],\n",
+    "                                  max_depth=1)\n",
+    "feature_matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03e03d97",
+   "metadata": {},
+   "source": [
+    "This feature matrix can be saved to disk or computed and brought into memory, using the appropriate Dask DataFrame methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19457b84",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm_computed = feature_matrix.compute()\n",
+    "fm_computed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af511e72",
+   "metadata": {},
+   "source": [
+    "While this is a simple example to illustrate the process of using Dask DataFrames with Featuretools, this process will also work with an `EntitySet` containing multiple dataframes, as well as with aggregation primitives.\n",
+    "\n",
+    "## Limitations\n",
+    "\n",
+    "The key functionality of Featuretools is available for use with a Dask `EntitySet`, and work is ongoing to add the remaining functionality that is available when using a pandas `EntitySet`. There are, however, some limitations to be aware of when creating a Dask `Entityset` and then using it to generate a feature matrix. The most significant limitations are reviewed in more detail in this section."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "a2212141",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    If the limitations of using a Dask ``EntitySet`` are problematic for your problem, you may still be able to compute a larger-than-memory feature matrix by partitioning your data as described in :doc:`performance`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f99e3d0",
+   "metadata": {},
+   "source": [
+    "### Supported Primitives\n",
+    "\n",
+    "When creating a feature matrix from a Dask `EntitySet`, only certain primitives can be used. Primitives that rely on the order of the entire DataFrame or require an entire column for computation are currently not supported when using a Dask `EntitySet`. Multivariable and time-dependent aggregation primitives also are not currently supported.\n",
+    "\n",
+    "To obtain a list of the primitives that can be used with a Dask `EntitySet`, you can call `featuretools.list_primitives()`. This will return a table of all primitives. Any primitive that can be used with a Dask `EntitySet` will have a value of `True` in the `dask_compatible` column."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c7410cef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "primitives_df = ft.list_primitives()\n",
+    "dask_compatible_df = primitives_df[primitives_df[\"dask_compatible\"] == True]\n",
+    "dask_compatible_df.head()\n",
+    "dask_compatible_df.tail()"
```

Comment:
Split into it's own code cell so we get the output from both the `head()` and `tail()` calls